src/megabot-internals/intervals.js

```diff
@@ -58,7 +58,7 @@ module.exports = [
           const member = await global.bot.guilds.get(ids.guild).getRESTMember(x.wb_id)
           const roles = member.roles.filter(x => rewards.includes(x))
           dlog(4, {
-            message: `About to remove roles from ${member.username}#${member.discriminator}. For reference later, they've accrued a total of ${x.properties.exp} EXP, and had ${roles.length} roles`
+            message: `About to remove roles from ${member.username}#${member.discriminator} \`\`${member.id}\`\`. For reference later, they've accrued a total of ${x.properties.exp} EXP, and had ${roles.length} roles`
```

Comment:
Is there a reason why there are 2 backticks?
```suggestion
            message: `About to remove roles from ${member.username}#${member.discriminator} (\`${x.wb_id}\`). For reference later, they've accrued a total of ${x.properties.exp} EXP, and had ${roles.length} roles`
```

Reply:
Nope I just thought u needed two for it to work for some reason.

Also thanks for the catch on the variable change, I just assumed since the rest of the message used member that I could also use it.